src/ai/backend/web/templates/config.toml.j2

```diff
@@ -24,6 +24,7 @@ connectionMode = "SESSION"
 {% toml_field "force2FA" config["service"]["force_2FA"] %}
 {% toml_field "systemSSHImage" config["service"]["system_SSH_image"] %}
 {% toml_field "directoryBasedUsage" config["service"]["directory_based_usage"] %}
+{% toml_field "maxCountForPreOpenedPort" config["session"]["max_count_for_preopen_ports"] %}
```

Comment:
Perhaps changes from [here](https://github.com/lablup/backend.ai-webui/pull/1861/files#diff-afc7c81ccfa8595a7b1c9b0199bc353d01316fd518c5295cdfbe8ebcc1556ee8R29) are not reflected at this PR?